include/silo/preprocessing/preprocessing_config_reader.h

```diff
@@ -17,13 +17,13 @@ struct OptionalPreprocessingConfig {
     */
    std::optional<std::filesystem::path> output_directory;
    /**
-    * The directory where the intermediate results will be stored
+    * The directory where temporary results will be stored
     * that are not relevant for an end user.
     */
-   std::optional<std::filesystem::path> intermediate_results_directory;
+   std::optional<std::filesystem::path> temp_directory;
```

Comment:
Please change accordingly in the LAPIS docs: https://github.com/GenSpectrum/LAPIS/blob/main/lapis2-docs/src/content/docs/maintainer-docs/references/preprocessing.mdx

Reply:
We can also make it part of https://github.com/GenSpectrum/LAPIS/issues/581

Reply:
Reverted it